### PR TITLE
About page

### PR DESCRIPTION
**Pull request by haowei99**

### CODE REVIEW

**Review by CalderLund — DISMISSED** [review]
LGTM. Still needs rest of team to add descriptions.

File: src/views/PageNotFoundPage.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
             ERROR: 404
         </div>
         <div class="text-h7">
-            The page that you are looking for does not exist. Clicking the following link to go back to the main page
+            The page that you are looking for does not exist. Clicking the following <a href='/'>link</a> to go back to the main page
```
Comment (kevinfwl) [inline src/views/PageNotFoundPage.vue:7]: Clicking the following... ->  Click the following

File: src/views/AboutPage.vue [hunk 2]
```diff
@@ -0,0 +1,196 @@
+<template>
+   <div style="background-color:ghostwhite; overflow-y: auto;">
+      <v-app-bar style="background: linear-gradient(38.92deg, #8198AB 36.61%, #2B435D 102.9%), #4A75AD;">
+         <router-link to="/">
+            <svg class="logo" width="40" height="55" viewBox="0 0 61 70" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (kevinfwl) [inline src/views/AboutPage.vue:5]: Is this the icon? I am thinking if we should put it into a separate component, might be cleaner

**Review by kevinfwl — COMMENTED** [review]
Some minor suggestions

File: src/views/PageNotFoundPage.vue (hunk 1 above)
Comment (haowei99) [reply src/views/PageNotFoundPage.vue:7]: Good catch

File: src/views/AboutPage.vue (hunk 2 above)
Comment (haowei99) [reply src/views/AboutPage.vue:5]: You are right. Let me organize it.

**Review by CalderLund — APPROVED** [review]
Looks good for now. Still need Liz's part.

**Pull request merged**